fe/fe-core/src/main/java/com/starrocks/persist/metablock/SRMetaBlockFooter.java

```diff
@@ -0,0 +1,18 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.persist.metablock;
+
+import com.google.gson.annotations.SerializedName;
+
+public class SRMetaBlockFooter {
+    @SerializedName(value = "c")
```

Comment:
```suggestion
    @SerializedName(value = "checksum")
```

Reply:
`{"c": 1236543}` is much shorter than `{"checksum": 1236543}`. It's just a token in json, not for people to read directly.